Riot/Utils/Tools.m

```diff
@@ -37,6 +37,9 @@ + (NSString *)presenceText:(MXUser *)user
                 break;
                 
             case MXPresenceUnknown: // Do like matrix-js-sdk
+                presenceText = @"";
```

Comment:
Could we return `nil` instead of empty string, possibly directly as `return nil`? An empty string might still be undergoing some of the `stringByAppendingString` processing that is happening below.

Also the `Do like matrix-js-sdk` comment is probably not accurate after this change anymore

Reply:
Thanks @Anderas, I made the requested changes.